rollup.config.js

```diff
@@ -0,0 +1,42 @@
+import builtins from 'rollup-plugin-node-builtins';
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import sourcemaps from 'rollup-plugin-sourcemaps';
+import pkg from './package.json';
+
+export default [
+    // browser-friendly UMD build
```

Comment:
As we're supporting `<script src="...">` imports, shouldn't we also bundle a minified version of the UMD file?

Reply:
We could, but the only reason I added this build is for StackBlitz support, I don't think people are going to use this package without a bundler.

Reply:
All right. Let's keep it like this until someone wants a minified file.